api-rest-common/src/main/java/edu/mit/ll/em/api/rs/RegisterUserNew.java

```diff
@@ -0,0 +1,146 @@
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package edu.mit.ll.em.api.rs;
+
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.hibernate.validator.constraints.Email;
+import org.hibernate.validator.constraints.NotBlank;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+public class RegisterUserNew extends APIBean {
+    @NotNull(message="Please provide Organization Type Id.")
+    private Integer organizationTypeId;
+    @NotNull(message="Please provide Organization Id.")
+    private Integer organizationId;
+    @NotBlank(message="Please provide First Name.")
+    private String firstName;
+    @NotBlank(message="Please provide Last Name.")
+    private String lastName;
+    @NotEmpty(message="Please provide valid Email Address.")
+    @Email(message="Please provide valid Email Address.")
+    private String email;
+    @Pattern(regexp="^(\\(\\d{3}\\) \\d{3}-\\d{4})?$", message="Please provide valid Phone number.")
+    private String phone;
+    @NotNull(message="Please provide Password of 8-20 characters with at least one digit, one upper case letter, one lower case letter and one special symbol @#$%-_!.")
+    @Pattern(regexp="^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@#$%_!-])[a-zA-Z0-9@#$%_!-]{8," +
+            "20}$",
+            message="Please provide Password of 8-20 characters with at least one digit, one upper case letter, one lower case letter and one special symbol @#$%-_!.")
+    private String password;
+    private Set<Integer> teams = new HashSet<Integer>();
+
+    public RegisterUserNew() {
+    }
+
+    public RegisterUserNew(Integer organizationTypeId, Integer organizationId, String firstName, String lastName, String email, String phone, String password, Collection<Integer> teams) {
```

Comment:
I believe @tparkergh mentioned, but, this appears to be a duplicate of RegisterUser.